main.tf

```diff
@@ -47,7 +41,7 @@ module "alb" {
         timeout             = var.alb_health_check_timeout
         matcher             = var.alb_health_check_valid_response_codes
         port                = "traffic-port"
-        protocol            = "HTTP"
+        protocol            = var.app_type == "web" ? "HTTP" : "TCP"
```

Comment:
(can't comment there)
How would `path                = var.alb_health_check_path` work? Should we use locals to set a path specific for TCP?